README.md

```diff
@@ -71,29 +100,46 @@ how the build was triggered from GitHub.
 
 Packer will use your
 [standard AWS environment](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html)
-to build the image.
+to build the image, however you will need to set up one profile for the
+previously-created build user and another profile to assume the associated
+`EC2AMICreate` role.  You will need the `aws_access_key_id` and
+`aws_secret_access_key` that you set as GitHub secrets earlier.
+
+Add the following blocks to your AWS credentials file (be sure to replace the
+dummy account ID in the `role_arn` with your own):
+
+```console
+[test-freeipa-server-packer]
+aws_access_key_id = AKIAXXXXXXXXXXXXXXXX
+aws_secret_access_key = XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+
+[cool-images-ec2amicreate-freeipa-server-packer]
+role_arn = arn:aws:iam::111111111111:role/EC2AMICreate-test-freeipa-server-packer
+source_profile = test-freeipa-server-packer
+role_session_name = example
+```
 
 The [Packer template](src/packer.json) requires two environment variables to be defined:
 
-- `BUILD_REGION`: the region in which to build the image.
-- `BUILD_REGION_KMS`: the kms key alias to use to encrypt the image.
+* `BUILD_REGION`: The region in which to build the image.
+* `BUILD_REGION_KMS`: The KMS key alias to use to encrypt the image.
 
 Additionally, the following optional environment variables can be used
 by the [Packer template](src/packer.json) to tag the final image:
 
-- `GITHUB_IS_PRERELEASE`: boolean pre-release status
-- `GITHUB_RELEASE_TAG`: image version
-- `GITHUB_RELEASE_URL`: URL pointing to the related GitHub release
+* `GITHUB_IS_PRERELEASE`: Boolean pre-release status
+* `GITHUB_RELEASE_TAG`: Image version
+* `GITHUB_RELEASE_URL`: URL pointing to the related GitHub release
```

Comment:
I think these descriptions should also end in periods.

Reply:
I made these changes in `skeleton-packer-cool` repo (see https://github.com/cisagov/skeleton-packer-cool/commit/afc9eaeef402ebae2829c366dbe0eac9dd99e008), so they will get pulled in to this repo at some point in the future.